v03_pipeline/lib/misc/lookup.py

```diff
@@ -98,15 +98,15 @@ def remove_family_guids(
             )
         ),
     )
-    project_i = ht.project_guids.index(
-        project_guid,
-    )  # double reference because new expression
+    ht = ht.filter(
+        hl.any(ht.project_stats.map(lambda fs: hl.any(fs.map(hl.is_defined)))),
+    )
     return ht.annotate_globals(
         project_families=hl.dict(
-            hl.enumerate(ht.project_families.items()).starmap(
-                lambda i, item: (
+            ht.project_families.items().map(
```

Comment:
I don't think `ht.project_families.items()` is guaranteed to return items in order... so this is more correct if less consistent with the rest of the code.